test/Interop/Cxx/templates/class-template-variadic.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+//
+// We can't yet call member functions correctly on Windows (SR-13129).
+// XFAIL: OS=windows-msvc
+
+import ClassTemplateVariadic
+import StdlibUnittest
+
+var TemplatesTestSuite = TestSuite("TemplatesTestSuite")
+
+TemplatesTestSuite.test("variadic-class-template") {
```

Comment:
Could you add a test for `Tuple` itself (not just `Pair`)?

Reply:
Added a failing test and a TODO. Swift currently sees it as if we passed too many template arguments.